### DISCUSSION TITLE
Как выводить метаполя в содержимом записи или в ушке?

### DISCUSSION THREAD

**Discussion by nvda-ru** [maxsite/cms#461] [Q&A]
Доброго времени суток!
Уважаемые специалисты, пожалуйста, научите, каким образом можно выводить метаполя в любом месте записи, а также в ушке.
Прочитал вот этот материал
https://max-3000.com/page/maxsite-cms-dlja-verstalshhikov-5

А также, на этой ↑ странице есть ссылка на материал с примерами по созданию каталога.
Что-то из этого смог повторить.
А именно добавить нужные метаполя.
Вывести их в коде шаблона: перед и после текста записи.
А вот, как выводить в любом месте содержимого записи: до или после [cut].
Выводить разные или повторяющиеся метаполя в нескольких местах одной записи, например: до [cut и после [cut].
Как сделать всё тоже самое, но при помощи ушек. Это чтоб не повторять php код внутри записи, а вставлять в нужные места ушки с готовым набором метаполей.
Если возможно, то покажите пожалуйста образцы кода.
Может где-то уже такое опубликовано.

У меня социальный проект для незрячих, делаю всё своими силами.
Давно назрел вопрос, создать каталог дополнений для программы экранного доступа NVDA, с информативными страницами по каждому дополнению. Сейчас каталог на одной странице, а количество дополнений перешагнуло за 400 штук, из-за чего страница стала просто огромной и тяжелой, а главное, неудобной для поиска нужного.
Но, вот выводить информацию из метаполей перед текстом записи и после текста записи, не совсем  то что нужно.
Для некоторых данных, буду и это использовать, но нужно сделать вывод данных в любом месте записи.

Буду весьма благодарен, если покажете примеры кода, чтобы реализовать такое, если конечно это не займёт у вас многих часов работы.
Благодарю.

**Comment by maxsite** [comment]
Метаполя принадлежат самой записи. Нет никакого смысла создавать метаполе, что-бы потом его выводить в тексте записи. Просто укажите нужные данные в самой записи.
